packages/ui-checkbox/src/Checkbox/__tests__/Checkbox.test.tsx

```diff
@@ -206,7 +206,7 @@ describe('<Checkbox />', async () => {
       const checkbox = await CheckboxLocator.find()
       const input = await checkbox.find('input')
 
-      await input.blur()
+      await input.focusOut()
```

Comment:
This change is because React 17 listens to the native focusIn and focusOut events to dispatch is own onFocus and onBlur events. https://reactjs.org/blog/2020/08/10/react-v17-rc.html#aligning-with-browsers

https://blog.saeloun.com/2021/05/14/react-17-uses-browse-focusin-focusout-for-onfocus-onblur